test/plugin/scenarios/solon-2.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,118 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+  - serviceName: solon-2.x-scenario
+    segmentSize: 3
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: Solon:HEAD:/testcase/healthCheck
```

Comment:
`/healthCheck` should not be included, as this is not a part of tracing test.

Reply:
Tests fail after deletion
![image](https://github.com/apache/skywalking-java/assets/33675646/491d835c-4100-42af-9928-bc4f53110ab2)

Reply:
Isn't this something you should fix? This should not affect test in any case. If it is, it means your tests have some issues.